lesson14/css/styles.css

```diff
@@ -0,0 +1,122 @@
+/*reset*/
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+html,
+body {
+	margin: 0;
+	padding: 0;
+}
+
+body {
+	font-family: sans-serif;
+	color: #666;
+	position: relative;
+	width: 100%;
+}
+
+h1 {
+	color: rgb(0, 119, 255);
+	text-align: center;
+	margin-bottom: 50px;
+}
+
+span {
+	display: block;
+	font-size: 20px;
+}
+
+a {
+	color: rgb(153, 198, 248);
+	font-size: 1.8rem;
+}
+
+.ly_loading {
+	width: 100%;
+	height: 100%;
+	background-color: #fff;
+}
+
+.el_loadingImg {
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+}
+
+.btn {
+	padding: 10px 20px;
+}
+
+.bl_btnArea {
+	background-color: royalblue;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	padding: 20px;
+}
+
+.bl_paragraph {
+	width: 100%;
+	background-color: #fff;
+	padding: 40px;
+}
+
+.bl_paragraph:nth-child(even) {
+	background-color: #f4f4f4;
+}
+
+/*Modal*/
+.bl_modal {
+	width: 80%;
+	max-width: 900px;
+	background-color: rgba(255, 255, 255, 1);
+	padding: 40px 20px;
+	position: absolute;
+	top: 10%;
+	left: 50%;
+	z-index: 100;
+	transform: translateX(-50%);
+	/* visibility: hidden;
+	opacity: 0;
+	transition: 0.3s opacity ease-in; */
+}
+
+.el_closeModalBtn {
```

Comment:
I have the impression that this writing style has been increasing recently.

The reason is that hyphens and underscores are used in naming conventions in FLOCCS and BEM.

A camel case is used because the use of hyphens in block names is indistinguishable from the naming convention hyphens.